src/random/random.ts

```diff
@@ -0,0 +1,13 @@
+export class Random {
+  public static number(): number {
+    return Math.random() * (10000 - -10000) + -10000;
```

Comment:
why having `- -` and `+ -`? It makes it weird for me but maybe I'm wrong

Reply:
So, you definitely know how Math.random works but I'll write down why I've followed this syntax.

- Math.random -> a number between 0 and 1
- 10000 - -10000 it's always 20000 but it indicates (max and min)
- -10000 it will add the min again to the value

Basically, let's say Math.random returns

- 1,  1 * 20000 => 20000 +- 10000 = 10000
- 0, 0*20000 => 0 +- 10000 = -10000

You can see the range of min and max.

I'll replace the number values with MIN and MAX constant